aws-opentelemetry-distro/src/amazon/opentelemetry/distro/llo_handler.py

```diff
@@ -279,6 +310,12 @@ def _collect_llo_attributes_from_span(self, span: ReadableSpan) -> Dict[str, Any
         # Collect from span events
         if span.events:
             for event in span.events:
+                # Check if event name itself is an LLO pattern (e.g., "gen_ai.user.message")
+                if self._is_llo_attribute(event.name):
+                    # Put all event attributes as the content as LLO in log event
+                    all_llo_attributes[event.name] = dict(event.attributes) if event.attributes else {}
+
+                # Also check traditional pattern - LLO attributes within event attributes
```

Comment:
How much backwards compatibility are we planning to support?

Reply:
For strands specifically, they changed to use OTel GenAI semantic convention. Hope there will be not much changes from OTel in future. We can decide how much later.